executor/simple.go

```diff
@@ -829,8 +835,11 @@ func (e *SimpleExec) executeDropUser(s *ast.DropUserStmt) error {
 		}
 		activeRoles := e.ctx.GetSessionVars().ActiveRoles
 		if !checker.RequestVerification(activeRoles, mysql.SystemDB, mysql.UserTable, "", mysql.DeletePriv) {
-			if s.IsDropRole && !checker.RequestVerification(activeRoles, "", "", "", mysql.DropRolePriv) {
-				return core.ErrSpecificAccessDenied.GenWithStackByArgs("DROP ROLE")
+			if s.IsDropRole {
+				if !checker.RequestVerification(activeRoles, "", "", "", mysql.DropRolePriv) &&
```

Comment:
Here is `&&`, that means if a user has the `CreateUser` privilege **or** `DropRole` privilege, the can execute the drop role operation.

Reply:
right, if a user has the CreateUser privilege or DropRole privilege, the can execute the drop role operation.